tools/distrib/python/submit.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import shutil
+import subprocess
+
+parser = argparse.ArgumentParser(
+    description='Submit the package to a PyPI repository.')
+parser.add_argument(
+    '--repository', '-r', metavar='r', type=str, default='pypi',
+    help='The repository to push the package to. '
+         'Ensure the value appears in your .pypirc file. '
+         'Defaults to "pypi".'
+)
+parser.add_argument(
+    '--identity', '-i', metavar='i', type=str,
+    help='GPG identity to sign the files with.'
+)
+parser.add_argument(
+    '--username', '-u', metavar='u', type=str,
+    help='Username to authenticate with the repository. Not needed if you have '
+         'configured your .pypirc to include your username.'
+)
+parser.add_argument(
+    '--password', '-p', metavar='p', type=str,
+    help='Password to authenticate with the repository. Not needed if you have '
+         'configured your .pypirc to include your password.'
+)
+args = parser.parse_args()
+
+# Move to the root directory of Python GRPC.
+pkgdir = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                      '../../../src/python/src')
+os.chdir(pkgdir)
```

Comment:
I'm sure I left a comment about this, but I can't find it now (nor the whole conversation it happened in) in GitHub's awful interface.

There is no need to do this os.chdir. Simply pass cwd=pkgdir to the two subprocess.call() calls below (and pass os.path.join(pkgdir, 'dist') to shutil.rmtree on line 39.)